ib_manifest_util/util.py

```diff
@@ -136,3 +142,83 @@ def download_files(urls: str | list) -> list:
         logger.info(f"Downloaded file {filename} from {address}.")
 
     return filenames
+
+
+def verify_local_channel_environments(
+    local_channel_env: str | Path,
+    offline_mode: bool = True,
+    conda_binary_loc: str | Path | None = None,
+):
+    """
+    Verify that the conda environment (`local_channel_env`) can be successfully created.
+
+    Args:
+        local_channel_env: path to local environment yaml
+        offline_mode: whether to run `conda env create` with the `--offline` flag, default is true
+        conda_binary_loc: path to a `conda` binary, default will try to locate the `conda` binary by using the return value of `$ which conda`
+
+    NOTE: It is assumed that the `conda-vendor vendor` command has already been run and that the `local_channel_env`
+    has a single channel set to `path/to/local_channel/folder`.
```

Comment:
`path/to/local_channel/folder`
seems like I've only seen this specified as `./path/to/folder` is the '.' important?

Reply:
I tried it with just `myenv` (where `myenv` was the folder with the channel data and it worked). I suspect though if you have a folder name that is the same as one of the online channels, (`default` or `conda-forge`), it will try to pull from those. Maybe an additional note is necessary.